include/flux/op/filter_map.hpp

```diff
@@ -0,0 +1,68 @@
+
+// Copyright (c) 2024 Tristan Brindle (tcbrindle at gmail dot com)
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef FLUX_OP_FILTER_MAP_HPP_INCLUDED
+#define FLUX_OP_FILTER_MAP_HPP_INCLUDED
+
+#include <flux/core.hpp>
+#include <flux/op/map.hpp>
+
+namespace flux {
+
+    namespace detail {
```

Comment:
Formatting: nested namespaces don't add a level of indentation

(I'll get round to adding a .clang-format config eventually...)